litestar/_openapi/schema_generation/plugins/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .dataclass import DataclassSchemaPlugin
+from .pagination import PaginationSchemaPlugin
+from .struct import StructSchemaPlugin
+from .typed_dict import TypedDictSchemaPlugin
+
+__all__ = ("openapi_schema_plugins",)
+
+# NOTE: The Pagination type plugin has to come before the Dataclass plugin since the Pagination
```

Comment:
There's a few cases where we rely on implicit ordering (same with middlewares). I think it might be worth it to add an explicit API for this, e.g. with some sort of priority configuration. Wdyt?

Reply:
Yeh I think that makes sense - this would be via the plugin registry I assume?